app/uk/gov/hmrc/tradergoodsprofiles/controllers/GetRecordsController.scala

```diff
@@ -68,11 +67,7 @@ class GetRecordsController @Inject() (
     lastUpdatedDate: Option[String]
   ): EitherT[Future, Result, Option[Instant]] =
     EitherT.fromEither[Future](
-      Try(lastUpdatedDate.map { dateTime =>
```

Comment:
I have removed the formatter as here we only need to check if this is a Zulu date, so Instant.parse shuold be enough. We can format this date in the router when we pass the date to EIS (which we already do)